packages/graph/hash_graph/tests/testdata/data_type/mod.rs

```diff
@@ -0,0 +1,7 @@
+pub const BOOLEAN_V1: &str = include_str!("boolean.json");
+pub const EMPTY_LIST_V1: &str = include_str!("empty-list.json");
+pub const NULL_V1: &str = include_str!("null.json");
+pub const NUMBER_V1: &str = include_str!("number.json");
+pub const OBJECT_V1: &str = include_str!("object_v1.json");
+pub const OBJECT_V2: &str = include_str!("object_v2.json");
+pub const TEXT_V1: &str = include_str!("text.json");
```

Comment:
Not important for this PR but when we come to writing more tests can we start to organise these better (the files themselves). For example I perhaps would have grouped a lot of the data types under `primitive/text.json` etc.

Not suggesting we do that as part of this PR though because I'm not sure we want the primitive data types sticking around here anyway.

Reply:
This is only test data and they are stored in `data_type/`. Why do you want to move them to `primitive`? The module is named after their type.

Reply:
Sorry I mean `data_type/primitive/number`. But anyway let's not do that for now. I'm just saying I expect us to put quite a lot more sample data within these directories, and we'll want to be able to group them so there isn't this big unwieldy flat list of files.

Reply:
Ah, yes, that makes sense. I think we should do so as soon as we add more 🙂